src/model/bpmn/shape/ShapeBpmnElement.ts

```diff
@@ -21,11 +21,17 @@ export default class ShapeBpmnElement {
 }
 
 export class ShapeBpmnEvent extends ShapeBpmnElement {
-  constructor(id: string, name: string, elementKind: ShapeBpmnElementKind, readonly eventKind: ShapeBpmnEventKind, parentId?: string) {
+  constructor(id: string, name: string, elementKind: ShapeBpmnElementKind, readonly eventKind: ShapeBpmnEventKind, parentId: string) {
     super(id, name, elementKind, parentId);
   }
 }
 
+export class ShapeBpmnBoundaryEvent extends ShapeBpmnEvent {
+  constructor(id: string, name: string, elementKind: ShapeBpmnElementKind, eventKind: ShapeBpmnEventKind, parentId: string, readonly isInterrupting: boolean = true) {
```

Comment:
:warning: as the elementKind should only be set to `EVENT_BOUNDARY`, I suggest to remove the constructor argument and to pass the value directly to the super constructor.
Notice that for `ShapeBpmnEvent` we should restrict the ShapeBpmnElementKind values that are accepted: only event related kinds